projects/angular/components/ui-grid/src/ui-grid.component.ts

```diff
@@ -781,4 +797,18 @@ export class UiGridComponent<T extends IGridDataEntry> extends ResizableGrid<T>
         }
         return `${this.selectionManager.isSelected(row) ? 'deselect' : 'select'} row ${this.dataManager.indexOf(row)}`;
     }
+
+    public isRowHeader(columns?: UiGridColumnDirective<T>[], index?: number) {
```

Comment:
When can this be called with no arguments?
```suggestion
    public isRowHeader(columns: UiGridColumnDirective<T>[], index: number) {
```